usr/share/rear/lib/layout-functions.sh

```diff
@@ -530,10 +530,7 @@ get_disk_size() {
 
     local block_size=$(get_block_size ${disk_name%/*})
 
-    [ -r /sys/block/$disk_name/size ]
-    BugIfError "Could not determine size of disk $disk_name, please file a bug."
-
-    local nr_blocks=$( < /sys/block/$disk_name/size)
+    local nr_blocks=$(retry_command "cat /sys/block/$disk_name/size")
```

Comment:
You remove line 534 with a specific error message that also explains what it was trying to do and replace that with a very generic error message that only says what it tried (the command itself) but not what the purpose was. Every use case will have a different explanation that should be show to the user.

Just a few imaginary use cases to illustrate this point:
```
retry_command ip a \| grep -q inet6
retry_command test -b /dev/mapper/foobar
```

If the error message only shows that info you won't know what ReaR was trying to do there or what the implication is or what you as a user could do to fix it. All that can be part of a custom error message.